### DISCUSSION TITLE
Issues in long time simulation of round jet

### DISCUSSION THREAD

**Discussion by MENG-light** [byuflowlab/FLOWVPM.jl#10] [Q&A]
I had set many parameters based on your doctoral thesis to simulate the jet flow, but I still cannot reproduce the results shown in your thesis. When I use the rVPM coupled SGS model to simulate, the simulation will break down before t = 20 ms due to the numerical instability. I cannot finish a long-time simulation like yours.
I want to know, for the simulation of a round jet, how to choose these parameters, such as overlap, dx/θ, the kernel function, how to set the parameters of FMM, whether or not to use the relaxation of vorticity, and whether or not to remove the particles far away from the exit. Especially, when I use RBF to fit the vorticity profile, the jet flow will be expansive rather than contractile.
The code that I used to simulate it is attached here. The code files (roundjet.jl, roundjet_functions.jl, and roundjet_simulation.jl) are the same as those in examples in FLOWVPM; the "run_roundjet.jl" file is a script that I use to run simulation. 
[round_jet_simulation_code.zip](https://github.com/byuflowlab/FLOWVPM.jl/files/14939424/round_jet_simulation_code.zip).

I would also be grateful for any suggestions that you could provide to further enhance my understanding. It would be best if you could provide a file of parameter settings for the simulation of the round jet that you have done.

**Comment by EdoAlvarezR** [comment]
Hi @MENG-light! I apologize for not circling back to this in so long. I just did some digging through my dissertation files and found what I think the driver was. Feel free to continue the conversation here: [LINK](https://github.com/byuflowlab/FLOWUnsteady/discussions/187)
